docker-deploy/generate_config.sh

```diff
@@ -87,7 +87,7 @@ GenerateConfig() {
 			sed -i "s#<party.id>#${party_id}#g" ./confs-$party_id/confs/eggroll/conf/eggroll.properties
 
 			#python env
-			sed -i "s#<venv>#${venv_dir}#g" ./confs-$party_id/confs/eggroll/conf/eggroll.properties
+			sed -i "s#<venv>##g" ./confs-$party_id/confs/eggroll/conf/eggroll.properties
```

Comment:
I think we can set value to "eggroll.properties" directly rather than update the file with this line.